csuntechlab/iSTART
# STR-5c -- FE: Initialize Vue3, VueRouter, VueX, Testing with Nuxt

# Description
'Write a meaningful description of your task'
  Initialize Vue3, VueRouter, VueX, Testing with Nuxt (didn't end up getting utilized), set up es lint +standard on commit 
# Testing
'What does the reviewer need to do to test your PR?'
1. You can simply go to `localhost:8080` to see the very very temp landing page
2. run `yarn run dev` to see everything work
3. run `yarn run test` to see jest work
4. Go back to the landing to click on the `about` and `home` links to see vue router work
5. Finally, go to a vue page:
 - make a bunch of line breaks
 - then STAY on the vue file, add and commit but DON'T PUSH
 - when you commit you'll see that our eslinter is working
# Installation
'Are there any dependencies the reviewer needs to download?'
- you will need to run `yarn install` since yarn is not in our container yet.
